src/test/java/org/apache/commons/collections4/ListUtilsTest.java

```diff
@@ -500,4 +500,45 @@ public void testSubtractNullElement() {
 
         assertEquals(expected, result);
     }
+    
+    @Test
+    public void testremoveNullElements() {
```

Comment:
The test name would have to be corrected to `testRemoveNullElements` (upper case `R` in remove)

Reply:
Yes, that was a typo unnoticed. Thanks a lot for pointing it out. Fixed it now.